nlp_profiler/noun_phase_count.py

```diff
@@ -1,20 +1,20 @@
-import en_core_web_sm
-import spacy
+from nltk.tokenize import word_tokenize
+from nltk.tag import pos_tag
 from nlp_profiler.constants import NaN
-nlp = en_core_web_sm.load()
+import re, emoji
 
+def gather_nouns(sentence: str):
 
-def gather_nouns(sent: str):
-
-    if not isinstance(sent, str) or len(sent) == 0:
+    if not isinstance(sentence, str) or len(sentence) == 0:
         return []
-    
-    doc = nlp(sent)
-    return [(x.text, x.label_) for x in doc.ents]
+    emoji_decoded = emoji.demojize(sentence, delimiters=("", "")).lower().strip() # Decoding Emoji's
```

Comment:
Would this affect the word/phrase count, in that case it's a good change.

Are emojis separated by a space a word? This is debatable, what do you think? I think a model as good as spacey should have this handled well.

Reply:
Decoded emoji are not space replaced, When I tried to separate the words it wasn't able to recognize all the words. 

For handling the noun phrase, I initially started with `spacy` but it was worked with Specific Country, person names. Not with the reference words. So switched to `NLTK pos tagger`.

Reply:
Great finding as I think the `count_words()` will eventually need some checking too, if what you say above, it can impact us in other places.

Reply:
Yes, It does impact. If we consider that as a use-case. Then for every sentence, we have to write a preprocess pipeline to remove all anomalies. 
Then only we can place the other engines.